### ISSUE TITLE
[GnoVM] Request: add simple way to get VM cycles on specific call

### ISSUE THREAD

**Issue by leohhhn** [gnolang/gno#1973]
## Description

It would be great if we can have a simple way to see how many cycles a specific call has spent. For example, adding a `--measure-cycles` flag to `gnokey` would be an idea to start with.

**Comment by deelawn** [comment]
The gas used should be a function of the cycles spent.  Would an end user care how many cycles are used?

**Comment by leohhhn** [comment]
Are we sure that gas will be a function of gas used? 😄

**Comment by deelawn** [comment]
Yes, it will be. Is your proposal to return VM cycles used like we return gas used when sending transactions via gnokey?

**Comment by leohhhn** [comment]
Correct!

**Comment by leohhhn** [comment]
For reference - it seems that the `gno test` binary already has an option for displaying cycles along with some other metrics - `gno test . -v -print-runtime-metrics`. This is an alright workaround, ie you can make a test simply calling the function you want to check cycles for:

```
func TestC(t *testing.T) {
	Render("")
}
```

Still, I would like to see a `-debug` or `-mertrics` option when using `gnokey`. WDYT?

**Comment by deelawn** [comment]
I'm not sure I'm convinced. Maybe someone else can chime in on this.
